feature-libs/organization/account-summary/components/account-summary-components.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { PageMetaResolver, provideDefaultConfig } from '@spartacus/core';
+import { AccountSummaryPageMetaResolver } from '@spartacus/organization/account-summary/core';
+import { accountSummaryCmsConfig } from './account-summary/account-summary.config';
+import { AccountSummaryDocumentModule } from './account-summary/details/document/account-summary-document.module';
+import { AccountSummaryHeaderModule } from './account-summary/details/header/account-summary-header.module';
+import { AccountSummaryListModule } from './account-summary/list/account-summary-list.module';
+
+@NgModule({
+  imports: [
+    AccountSummaryListModule,
+    AccountSummaryHeaderModule,
+    AccountSummaryDocumentModule,
+  ],
+  declarations: [],
+  providers: [
+    provideDefaultConfig(accountSummaryCmsConfig),
```

Comment:
maybe move this `accountSummaryCmsConfig` to list module. It configs `ManageAccountSummaryListComponent`.